lib/src/controls/navigation/navigation_view/view.dart

```diff
@@ -591,10 +603,10 @@ class NavigationViewState extends State<NavigationView> {
               paneResult = content;
           }
         }
-      } else {
+      } else if (widget.content != null) {
         paneResult = Column(children: [
           appBar,
-          // Expanded(child: widget.content),
+          Expanded(child: widget.content!),
         ]);
       }
```

Comment:
We would never reach this, since either pane or content must be provided, but it's good to add some safety.

```suggestion
      } else {
        throw 'Either pane or content must be provided';
      }
```